### DISCUSSION TITLE
`通过 reducers, effects 和 subscriptions 组织 model，简化 redux 和 redux-saga 引入的概念.` 是什么意思？是指的用model直接就简化redux和redux-saga的使用吗？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [umijs/umi#10411] [Q&A] [answered]
>elm 概念
通过 reducers, effects 和 subscriptions 组织 model，简化 redux 和 redux-saga 引入的概念.

请问如何理解这段话？
1.什么是elm？
2.`通过 reducers, effects 和 subscriptions 组织 model，简化 redux 和 redux-saga 引入的概念.` 是什么意思？是指的用model直接就简化redux和redux-saga的使用吗？

**Comment by fz6m** [comment, marked as answer]
See #10410 ，不要再使用 dva 了。
